src/services/solarPanel.ts

```diff
@@ -128,20 +125,20 @@ export function getPeakPower() {
 
 export async function solarPanelGridToSolarPanelModel(
   rennesApp: RennesApp,
-  matrixGrid: Matrix,
-  solarPanelGrids: SolarPanelGrid[],
+  usableIds: Matrix,
   orientation: string,
   roofInclinaison: number = 45,
   roofAzimut: number = 0
 ) {
   const solarPanelModels: SolarPanelModel[] = []
   const positions: number[][] = []
-  for (let i = 0; i < solarPanelGrids.length; i++) {
-    const spg = solarPanelGrids[i]
-    const index = i
-
-    const center = getSolarPanelGridCenter(matrixGrid, spg)
+  let index: number = 0
+  let offset = HeightOffset
+  if (isRoofFlat()) {
+    offset += 0.2
```

Comment:
Could you give a comment about the 0.2 value?

Reply:
Hmm, the heightoffset is from me I guess. But in this part, you add more 0.2. Is that correct?